test/functional/feature_rbf.py

```diff
@@ -627,6 +630,24 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+    def test_replacement_relay_fee(self):
+        outpoint = make_utxo(self.nodes[0], int(1.1 * COIN))
+
+        initial_transaction = CTransaction()
+        initial_transaction.vin = [CTxIn(outpoint, nSequence=0)]
+        initial_transaction.vout = [CTxOut(1 * COIN, DUMMY_P2WPKH_SCRIPT)]
+        initial_transaction_hex = initial_transaction.serialize().hex()
+        self.nodes[0].sendrawtransaction(initial_transaction_hex, 0)
+
+        # Equal fee, but the replacement does not provide a relay
```

Comment:
Is is "equal fee" if the fee is increased?

Again, I'd recommend to use MiniWallet, which will simplify this test a lot:

```py
    def test_replacement_relay_fee(self):
        wallet = MiniWallet(self.nodes[0])
        wallet.scan_blocks(start=77, num=1)
        tx = wallet.send_self_transfer(from_node=self.nodes[0])['tx']
        tx.vout[0].nValue -= 1
        assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())

Reply:
Ooops, leftover from the multiple updates of this PR with the converation [here](https://github.com/bitcoin/bitcoin/pull/22310#discussion_r657295787).